api/controllers/document.js

```diff
@@ -399,8 +394,8 @@ exports.protectedOpen = function (args, res, next) {
   assignIn(query, { "_schemaName": "Document" });
 
   Utils.runDataQuery('Document',
-    args.swagger.params.auth_payload.client_roles,
-    args.swagger.params.auth_payload.idir_user_guid,
+    ['public'],
```

Comment:
is this the plan for future to hardcode this, or will they being going back into an object?

Reply:
For now this will be hard-coded to only open public documents(don't require authentication to see). Normally the app has separate controllers for public/protected data but it looks like the app has always used this one controller for both. Not sure why that is, but I'm hard-coding for now to minimize the impact of a bug here. I'll spend more time to look into if/how to implement a dedicated "public" controller if necessary.